microscopic-monks/primal/screens/death_screen.py

```diff
@@ -0,0 +1,43 @@
+from primal.engine.screen import Screen
+from primal.engine.camera import OrthographicCamera
+from primal.engine.sprite import ColorSprite, Text
+
+
+class DeathScreen(Screen):
+    VP_WIDTH = 1280
```

Comment:
could the WIDTH and HEIGHT come from the parent? or maybe previously defined to keep the code DRY?

Reply:
They could, but they are not defined in the parent and they don't have to be the same over all screens. They are just "resolutions" of the camera, that is related only to the specific screen.

Reply:
a comment describing this fact would then make it very clear.